config/config.go

```diff
@@ -8,3 +8,6 @@ var SiteDomain = "ons.gov.uk"
 
 // PatternLibraryAssetsPath is the URL to the CSS and JS assets from the pattern library
 var PatternLibraryAssetsPath = "//cdn.ons.gov.uk/sixteens/ac0073c"
+
+// SupportedLanguages is a list of languages that are supported
+var SupportedLanguages = [2]string{"en", "cy"}
```

Comment:
Didn't you create constants in `go-ns` for the languages? If so does it make sense to use them?

Reply:
We added `LangEN="en"` and `LangCY="cy"` to go-ns which serves as a single place for us to change the LocaleCode used (personally think we should switch to [Unicode Technical Standard #35](Unicode Technical Standard #35) rather than current or ISO. 

This list is used as a single place to store all supported languages, to act as a single place to change the languages we support, at current we have nothing in place for this. I have however moved this to go-ns common package under the locale .go file I think this addresses the comment as now it can be vendored into any package that may use it.